test/NuGet.Core.Tests/NuGet.PackageManagement.Test/NuGetPackageManagerTests.cs

```diff
@@ -7034,6 +7035,230 @@ public async Task TestPacMan_PreviewInstallPackage_BuildIntegrated_NullVersion_T
             }
         }
 
+        [Fact]
```

Comment:
Given that the concept of primary and secondary sources exists, it's a good idea that we add some tests for that.

Reply:
Added 1 test for it:

https://github.com/NuGet/NuGet.Client/blob/dev-eryondon-NameSpacePackageConfig-InstallUpdate/test/NuGet.Core.Tests/NuGet.PackageManagement.Test/NuGetPackageManagerTests.cs#L7218-L7321#L7218-L7225